src/core/SampleBuffer.cpp

```diff
@@ -306,27 +306,13 @@ void SampleBuffer::convertIntToFloat ( int_sample_t * & _ibuf, f_cnt_t _frames,
 
 	// if reversing is on, we also reverse when
 	// scaling
-	if( m_reversed )
+	int idx = m_reversed ? ( _frames - 1 ) * _channels : 0;
+	for( f_cnt_t frame = 0; frame < _frames;
+					++frame )
 	{
-		int idx = ( _frames - 1 ) * _channels;
-		for( f_cnt_t frame = 0; frame < _frames;
-						++frame )
-		{
-			m_data[frame][0] = _ibuf[idx+0] * fac;
-			m_data[frame][1] = _ibuf[idx+ch] * fac;
-			idx -= _channels;
-		}
-	}
-	else
-	{
-		int idx = 0;
-		for( f_cnt_t frame = 0; frame < _frames;
-						++frame )
-		{
-			m_data[frame][0] = _ibuf[idx+0] * fac;
-			m_data[frame][1] = _ibuf[idx+ch] * fac;
-			idx += _channels;
-		}
+		m_data[frame][0] = _ibuf[idx+0] * fac;
+		m_data[frame][1] = _ibuf[idx+ch] * fac;
+		m_reversed ? idx -= _channels : idx += _channels;
```

Comment:
I'm not a fan of this style honestly. I don't think it's as easy to read as say:
```suggestion
		idx = m_revsered ? idx - _channels : idx + _channels;
```
or even
```suggestion
		idx += m_reversed ? -_channels : _channels;
```
Curious what everyone else thinks.

Reply:
I think I prefer the first suggestion too: `idx = m_reversed ? idx - _channels : idx + _channels;`

Reply:
Apart from being a tad bit longer, this isn't an issue. I'll change this.